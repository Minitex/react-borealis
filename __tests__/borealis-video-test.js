import React from 'react';
import renderer from 'react-test-renderer';
import BorealisVideo from '../src/borealis-video-player';

test('Borealis video', () => {
  const config = {
    video: {
      thumbnail: 'http://reflections.mndigital.org/utils/getthumbnail/collection/p16022coll38/id/0',
      transcript: {
        texts: [],
        label: 'Video',
      },
      height: 500,
      width: 500,
      src: 'http://reflections.mndigital.org/utils/getstream/collection/stc/id/8470'
    },
  };

  const component = renderer.create(
    <BorealisVideo config={config} />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
