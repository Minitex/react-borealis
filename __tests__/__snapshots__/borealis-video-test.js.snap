exports[`test Borealis video 1`] = `
<video
  controls={true}
  height={800}
  width={800}>
  <source
    src={undefined}
    type="video/mp4" />
  Your browser does not support the video tag.
</video>
`;
